### ISSUE TITLE
[COGNITION]: STEM Standalone - Home phone number SHOULD be formatted like mobile number on the review form page

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#12340]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The home phone number and the mobile phone number should be formatted using the parens for zip code and the dash between the first three and last four numbers. This is read out properly by screen readers and helps users of assistive tech. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Both phone numbers are formatted using the accepted telephone styling
- [ ] Both phone numbers announce themselves the same to assistive technology

## Environment

* https://staging.va.gov/education/apply-for-education-benefits/application/10203/review-and-submit

## Screenshots or Trace Logs

![Screen Shot 2020-08-10 at 10 11 15 PM](https://user-images.githubusercontent.com/934879/89853487-22296f80-db57-11ea-974d-86ded1f5899a.png)

**Comment by desireeturner** [comment]
@1Copenut As discussed, this issue appears to be form system-wide.

**Comment by 1Copenut** [comment]
@meganhkelley This was flagged in my staging review for BAH's STEM standalone form. @desireeturner let me know this appears to be a forms lib issue, so I've reassigned to you. LMK if I need to post it elsewhere.

**Comment by 1Copenut** [comment]
@meganhkelley This was flagged in my staging review for BAH's STEM standalone form. @desireeturner let me know this appears to be a forms lib issue, so I've reassigned to you. LMK if I need to post it elsewhere.
